# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1437855720 14400
#      Sat Jul 25 16:22:00 2015 -0400
# Node ID 1bc95d7148b729f1d2a8f098a80c7c9e8572dc74
# Parent  4e52267743d736edb94c256893b2d01c78ad2ffc
build: Fix installation of icons from source tarball

* etc/module.mk (install-icons): Remove redundant "etc/icons" from path
to icons when building from source tarball.

diff --git a/etc/module.mk b/etc/module.mk
--- a/etc/module.mk
+++ b/etc/module.mk
@@ -99,17 +99,17 @@ uninstall-local: uninstall-icons
 
 install-icons:
 	-if test -n "$(DESKTOP_FILE_INSTALL)"; then \
 	  $(DESKTOP_FILE_INSTALL) --dir=$(DESTDIR)$(datadir)/applications \
 	    --vendor $(VENDOR) etc/icons/octave.desktop; \
 	fi
 	for f in $(BUILT_PNG_ICONS); do \
 	  size=`echo $$f | $(SED) -n -e "s/.*-\([0-9]\+\)\.png/\1/p"`; \
-	  if test -f $$f; then d=; else d="$(srcdir)/etc/icons/"; fi; \
+	  if test -f $$f; then d=; else d="$(srcdir)/"; fi; \
 	  $(MKDIR_P) $(DESTDIR)$(datadir)/icons/hicolor/$${size}x$${size}/apps; \
 	  $(INSTALL_DATA) "$$d$$f" $(DESTDIR)$(datadir)/icons/hicolor/$${size}x$${size}/apps/octave.png; \
 	done
 	$(MKDIR_P) $(DESTDIR)$(datadir)/icons/hicolor/scalable/apps
 	$(INSTALL_DATA) $(srcdir)/etc/icons/octave-logo.svg $(DESTDIR)$(datadir)/icons/hicolor/scalable/apps/octave.svg
 	$(MKDIR_P) $(DESTDIR)$(datadir)/appdata
 	$(INSTALL_DATA) etc/icons/octave.appdata.xml $(DESTDIR)$(datadir)/appdata/$(VENDOR)-octave.appdata.xml
 
